draft-ietf-quic-transport.md

```diff
@@ -678,6 +691,22 @@ packet protection in detail.  After decryption, the plaintext consists of a
 sequence of frames, as described in {{frames}}.
 
 
+## Compound Packets {#packet-compound}
+
+A sender can combine multiple QUIC packets (typically a Cryptographic Handshake
+packet and a Protected packet) into one UDP packet, thereby reducing the number
+of UDP packets required to be emitted when application data can be sent during
```

Comment:
End the sentence at the comma.  Then "This can reduce the number of UDP datagrams that are needed to complete the handshake when multiple types of packets are sent at the same time."

Reply:
Done in 5ed67ea.